packages/compiler/test/config/config.test.ts

```diff
@@ -15,8 +16,24 @@ const scenarioRoot = resolvePath(
 
 describe("compiler: config file loading", () => {
   describe("file discovery", () => {
-    const loadTestConfig = async (path: string, errorIfNotFound: boolean = true) => {
-      const fullPath = join(scenarioRoot, path);
+    const loadTestConfig = async (
+      path: string,
+      lookup: boolean = true,
+      errorIfNotFound: boolean = true
+    ) => {
+      let fullPath = join(scenarioRoot, path);
+      try {
+        const stats = await fs.stat(fullPath);
+        const isDirectory = stats.isDirectory();
+
+        if (lookup === false && isDirectory) {
+          fullPath = join(fullPath, "tspconfig.yaml");
```

Comment:
do we need to do that? isn't `loadTypeSpecConfigForPath` already dealing with this?

Reply:
It does that if you allow it to go and recurse through folders, but since we don't want that behavior, the simplest solution to having to plumb the distinction through a bunch of methods is to simply turn the folder path here into a file path.

Reply:
I guess I am confused, specially with the latest change which seems to have removed all the lookup disabling behavior

Reply:
Maybe what needs to be tested for that is the `resolveCompilerOptions` instead of `loadConfigAtPath` as this is what really has the logic for resolving what config to load from which options you pass in the cli